src/quantum_siren/pipelines/training/ansaetze.py

```diff
@@ -66,6 +66,24 @@ def circuit_18(params: torch.Tensor | np.ndarray):
                 ],
             )
 
+    @staticmethod
+    def strongly_entangling_layers(params: torch.Tensor | np.ndarray):
+        """Encoding of two dimensional data using RX and RY gates.
+        The input is repeated across all qubits (vertically),
+        as specified by the shape of the input.
+
+        Args:
+            params (torch.Tensor | np.ndarray): Input data with the first value
+            parameterizing the RX gate and the second value parameterizing the RY gate.
+            Expects form to be [n_qubits, 3]
+        """
+        if params is None:
+            return 3
```

Comment:
What is up with this return value?

Reply:
This return value is used during the initialization of the weight vector. Because each ansatz has a variable number of parameters per layer, the idea is to call the ansatz without specific parameters and it returns the required number of parameters per layer.
But maybe we can drop that anyway because the init methods have changed.. let me check..

Reply:
resolved, now the redundant init is removed, thus no need for the hacky return value